qiskit_superstaq/compiler_output.py

```diff
@@ -58,11 +56,27 @@ def __repr__(self) -> str:
     def __eq__(self, other: Any) -> bool:
         if not isinstance(other, CompilerOutput):
             return False
+
+        if self.has_multiple_circuits() != other.has_multiple_circuits():
+            return False
+        elif self.has_multiple_circuits():
+            return (
+                self.circuits == other.circuits
+                and self.pulse_sequences == other.pulse_sequences
+                and self.jaqal_programs == other.jaqal_programs
+                and self.pulse_lists == other.pulse_lists
```

Comment:
my bad, should have included this in my previous suggestion:
```suggestion
                and self.pulse_lists == other.pulse_lists
                and self.seq == other.seq
```

Reply:
Done. Thanks.